src/deepseek_reverse/api.py

```
@@ -123,3 +137,119 @@ def generate():
+        )
+
+        async def generate() -> AsyncIterator[str]:
+            async for line in response.aiter_lines():
+                assert isinstance(line, bytes)
+                line = line.decode()
+                if "{" in line:
+                    chunk = json.loads(line.split(":", 1)[1])
+                    try:
+                        yield cast(str, chunk["choices"][0]["delta"]["content"])
+                    except:
+                        pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)